pwndbg/commands/hexdump.py

```diff
@@ -32,13 +33,17 @@
 def hexdump(address=None, count=pwndbg.config.hexdump_bytes):
     address = int(address)
     address &= pwndbg.arch.ptrmask
-    count   = int(count)
+    count   = max(int(count), 0)
```

Comment:
Does this break functionality of `hexdump A B` to hexdump `[A..B]` given that addresses may be interpreted as negative?

```
pwndbg> hexdump $rsp $rsp+10
+0000 0x7fffffffdc10  01 00 00 00  00 00 00 00  27 df ff ff  ff 7f 00 00  │....│....│'...│....│
+0010 0x7fffffffdc20
```